quickstarts/dockerdesktop.md

```diff
@@ -0,0 +1,105 @@
+Docker Desktop comes includes a standalone Kubernetes server and client, designed for local testing of Kubernetes.
+The Kubernetes cluster is started as a single-node cluster within a Docker container on your local system.
+
+# Installing the dependencies
+
+This quickstart assumes that you have the latest version of Docker Desktop, which you can get [here](https://docs.docker.com/desktop/).
+
+If you are running on Linux you'll need to install the `kubectl` binary separately, which you can get by following the instructions [here](https://kubernetes.io/docs/tasks/tools/).
+
+Once you have all the binary installed, make sure everything works:
+
+```shell
+# Validate kubectl if on Linux
+kubectl version
+```
+
+# Starting Kubernetes cluster
+
+This will start a local development cluster of Kubernetes with [Docker Desktop](https://docs.docker.com/desktop/kubernetes/) which runs in a container on your local machine.
+
+1. From the Docker Dashboard, select the Setting icon, or Preferences icon if you use a macOS.
+1. Select Kubernetes from the left sidebar.
+1. Next to Enable Kubernetes, select the checkbox.
+1. Select Apply & Restart to save the settings and then click Install to confirm.
+   This instantiates images required to run the Kubernetes server as containers, and installs the `/usr/local/bin/kubectl` command on your machine.
+
+
+# Applying Strimzi installation file
```

Comment:
I've suggested "Deploy Strimzi using installation files"

In terms of whether they should be steps with numbers I think they could but since there is a lot of text in between about the namespace perhaps that wouldn't help readability?